URA-695/reproducibility.py

```
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
```

Ruff (0.8.0)

Shebang is present but file is not executable

(EXE001)